internal/server/v1/firehose/firehose.go

```diff
@@ -33,6 +33,7 @@ func Routes(r *mux.Router, client entropyv1beta1.ResourceServiceClient, shieldCl
 	// read APIs
 	r.Handle("/projects/{projectSlug}/firehoses", handleListFirehoses(client, shieldClient)).Methods(http.MethodGet)
 	r.Handle("/projects/{projectSlug}/firehoses/{urn}", handleGetFirehose(client)).Methods(http.MethodGet)
+	r.Handle("/projects/{projectSlug}/firehoses/{urn}/history", handleGetFirehoseHistory(client)).Methods(http.MethodGet)
```

Comment:
what if we changed this to `/revisions`?

Reply:
Currently the datlantis frontend calls it history hence wanted to keep the name same as that.
Plus there is a small logic b/w the entropy revisions API and this API hence wanted to make sure that is expressed in some manner.